src/main/java/mate/academy/Main.java

```diff
@@ -1,23 +1,28 @@
 package mate.academy;
 
 import mate.academy.exception.AuthenticationException;
+import mate.academy.model.Order;
 import mate.academy.model.User;
 import mate.academy.service.AuthenticationService;
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
The task requires configuring the logger with appenders for both File and Console. Ensure that this configuration is implemented to meet the task's requirements.